api/v1alpha3/condition_types.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ANCHOR: ConditionSeverity
+
+// ConditionSeverity expresses the severity of a Condition Type failing.
+type ConditionSeverity string
+
+const (
+	// ConditionSeverityError specifies that a condition with Status=False represents
+	// an error about in process of transitioning to to the True state.
+	ConditionSeverityError ConditionSeverity = "Error"
+
+	// ConditionSeverityWarning specifies that a condition with Status=False represents
+	// a warning about in process of transitioning to to the True state.
+	ConditionSeverityWarning ConditionSeverity = "Warning"
+
+	// ConditionSeverityInfo specifies that a condition with Status=False represents
+	// a information about the process of transitioning to to the True state.
+	ConditionSeverityInfo ConditionSeverity = "Info"
+
+	// ConditionSeverityNode should apply only to condition is in state "True".
+	ConditionSeverityNone ConditionSeverity = ""
+)
+
+// ANCHOR_END: ConditionSeverity
+
+// ANCHOR: ConditionType
+
+// ConditionType is a valid value for ClusterCondition.Type.
+type ConditionType string
+
+// Common ConditionTypes used by Cluster API objects.
+const (
+	// ReadyCondition defines the Ready condition type that summarizes the operational state of a Cluster API object.
+	ReadyCondition ConditionType = "Ready"
+)
```

Comment:
Would be great to move these to `condition_consts.go`

Reply:
done!